src/anthropic/lib/bedrock/_client.py

```diff
@@ -116,6 +117,8 @@ def __init__(
 
         self.aws_access_key = aws_access_key
 
+        session = boto3.Session()
+        aws_region = session.region_name
```

Comment:
This is almost what we want! Ideally we'd want to set `aws_region` in order of specificity: 
1. specific region passed as argument
2. specific region passed as environment variable
3. specific region within config
4. default to us-east-1
 
however, as written in this PR, this would be:

1. specific region within config
2. specific region passed as argument
3. specific region passed as environment variable
4. default to us-east-1

if you could correct the ordering of assignment, that would be wonderful.

Reply:
@JDKardia, I fixed the issue here at #669, please review and merge